test/parallel/test-cluster-dgram-reuse.js

```diff
@@ -36,5 +36,7 @@ function close() {
     cluster.worker.disconnect();
 }
 
-for (let i = 0; i < 2; i++)
-  dgram.createSocket({ type: 'udp4', reuseAddr: true }).bind(common.PORT, next);
+const socket = dgram.createSocket({type: 'udp4', reuseAddr: true})
+  .bind(0, common.mustCall(() => {
+    socket.bind(this.address().port, next);
```

Comment:
This runs bind twice on the same socket but the previous test creates two separate sockets.